src/decode/lzma.rs

```diff
@@ -95,6 +113,9 @@ where
     LZB: lzbuffer::LZBuffer<W>,
 {
     _phantom: PhantomData<W>,
+    // buffer input data here if we need more for decompression, 20 is the max
+    // number of bytes that can be consumed during one iteration
+    tmp: std::io::Cursor<[u8; MAX_REQUIRED_INPUT]>,
```

Comment:
"tmp" isn't really meaningful. What does this contain? Maybe `partial_input_buf` would be clearer?